spark/src/test/scala/geotrellis/spark/summary/polygonal/MaxDoubleSpec.scala

```diff
@@ -114,19 +128,21 @@ class MaxDoubleSpec extends FunSpec with TestEnvironment with TestFiles {
   }
 
   describe("Max Double Zonal Summary Operation (collections api)") {
-    val inc = IncreasingTestFile.toCollection
```

Comment:
Each of these test classes have a section dedicated to `Summary Operation (collections api)` where they take a `TileLayerRDD[SpatialKey] with Metadata` and convert it to a `geotrellis.spark.ContextCollection`. Should we be considering an alternate `CollectionPolygonalSummary` method to go along with `RDDPolygonalSummary` to support the collections API? Or can we just delete all these tests and assume that users can lift their collections into an RDD if they want to perform polygonal summary operations?

If the path forward isn't super clear, I might make the argument for dropping support, writing up an issue, and re-adding it post 3.0 if it becomes clear that polygonal summary support in the collections API is very valuable to have.

Reply:
Forgot about those. This is the case where having a type class that abstracts over `RDD[(K, V)]` and `Seq[(K, V)]` would be super handy dandy and I loath repeating the mistake of just writing more code that side-steps this abstraction. Lets talk about this in person briefly and see what's reasonable.

Reply:
I created https://github.com/locationtech/geotrellis/issues/3004 to address this in the future as per our earlier discussion.